### PR TITLE
Update README.md

### LINKED ISSUE #1: Create README.md
It'll contain the following phases:-

- [x] Project Description
- [x] Links
- [x] Problem
- [x] Solution
- [x] User Journey
- [x] User Stories
- [x] Technology used
- [x] Clone it
- [x] Team Leader
- [x] Team members

### PR DESCRIPTION
**Pull request by zaher-aa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
 # FlowX
-Know weather for any where in the glob.
+## ✧ Project Description
+The main purpose for the project is give the user information about his currant weather,also alow him to search for any country weather.
```
Comment (AseelL) [inline README.md]: fix typos
giving*
current* 
allowing*

**Review by AseelL — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zaher-aa) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1,39 +1,50 @@
...
+- Responsive design.
+- Accurate information.
+- Autocomplete search field.
+- Clear landing page with slider.
+
+## ✧ Used Technologies
+- **HTML** → Markup.
+- **CSS** → Markup Styling.
+- **JavaScript** → DOM Manipulation.
+- **GitHub** → Deploy & Code Sotre.
```
Comment (AseelL) [inline README.md]: store

**Review by husamkamal — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,39 +1,50 @@
...
 
 ## ✧ User Journey
 - Opens the application and views a greeting screen with the app name
 - Clicks a button to move to the main page
 - Clicks “Allow” in the location permission prompt to give access to the website to know your location and view the weather situation in user’s area.
-- Views the weather in his current location
+- Views the weather in user's current location
 - Can scroll down to find more information about wind, humidity, UVI, and pressure.
-- Can click a search icon to search for other cities and view weather at them
+- Can click a search icon to search for other cities and view whether other places whether situations.
```
Comment (AseelL) [inline README.md]: weather , on both occasions

**Review by AseelL — CHANGES_REQUESTED** [review]

**Review by AseelL — APPROVED** [review]
yaaay ✨

**Review by husamkamal — APPROVED** [review]

**Pull request merged**